### PR TITLE
dual_cut_zero_tol

### LINKED ISSUE #81: error in conic/Mosek
I ran the file TEST_contFiniteExp.jl in our dropbox, with the new atom in Convex "sgeomean_eig". got this error:
```
LoadError: Mosek.MosekError(1390,"The lower bound specified is not a number (nan).")
 in putconboundslice at /Users/chris/.julia/v0.4/Mosek/src/msk7_functions.jl:2895
 in loadproblem! at /Users/chris/.julia/v0.4/Mosek/src/MosekConicInterface.jl:347
 in loadproblem! at /Users/chris/.julia/v0.4/Mosek/src/MosekConicInterface.jl:93
 in loadConicModel at /Users/chris/.julia/v0.4/Pajarito/src/conic.jl:516
 in coniccallback at /Users/chris/.julia/v0.4/Pajarito/src/conic.jl:968
 in optimize! at /Users/chris/.julia/v0.4/Pajarito/src/conic.jl:1172
 in solve! at /Users/chris/.julia/v0.4/Convex/src/solution.jl:32
 in solve! at /Users/chris/.julia/v0.4/Convex/src/solution.jl:13
 in dOpt at /Users/chris/Dropbox/misdp/julia/contFiniteExp.jl:37
 in include_string at loading.jl:266
 in include_string at /Users/chris/.julia/v0.4/CodeTools/src/eval.jl:32
 in anonymous at /Users/chris/.julia/v0.4/Atom/src/eval.jl:84
 in withpath at /Users/chris/.julia/v0.4/Requires/src/require.jl:37
 in withpath at /Users/chris/.julia/v0.4/Atom/src/eval.jl:53
 [inlined code] from /Users/chris/.julia/v0.4/Atom/src/eval.jl:83
 in anonymous at task.jl:58
while loading /Users/chris/Dropbox/misdp/julia/contFiniteExp.jl, in expression starting on line 80
```

### PR DESCRIPTION
**Pull request by emreyamangil**
Closes #81 

@chriscoey set dual_cut_zero_tol to 1e-8

### CODE REVIEW

File: src/conic.jl [hunk 1]
```diff
@@ -52,6 +52,7 @@ type PajaritoConicModel <: MathProgBase.AbstractConicModel
     instance::AbstractString    # Path to instance
     enable_sdp::Bool            # Indicator for enabling sdp support
     force_primal_cuts::Bool     # Enforces primal cutting planes under conic solver
+    dual_cut_zero_tol           # Tolerance to check if a conic_dual is zero or not
```
Comment (mlubin) [inline src/conic.jl]: ``::Float64``

File: src/conic.jl [hunk 2]
```diff
@@ -682,6 +684,9 @@ end
 
 function addDualCuttingPlanes!(m, mip_model, separator, cb, mip_solution)
 
+    zero_ind = filter(i->abs(separator[i]) < m.dual_cut_zero_tol, 1:length(separator))
```
Comment (mlubin) [inline src/conic.jl]: This creates two unnecessary temporary arrays, why not just use a ``for`` loop?

**Comment by mlubin** [comment]
:+1:

**Comment by emreyamangil** [comment]
Hopefully fixes our SDP issues!!

**Comment by chriscoey** [comment]
ok tell me when to pull. and i git checkout your fork emre? or is this
getting merged?

**Comment by emreyamangil** [comment]
getting merged! waiting appveyor!

**Pull request merged**